### PR TITLE
Hotkeys: Fix missing hotkey strings

### PR DESCRIPTION
**Pull request by degasus**
Also some cleanups to support more than 3 groups.

### CODE REVIEW

File: Source/Core/Core/HotkeyManager.cpp [hunk 1]
```diff
@@ -109,6 +109,8 @@ const std::string hotkey_labels[] =
 	_trans("Load State Last 6"),
 	_trans("Load State Last 7"),
 	_trans("Load State Last 8"),
+	_trans("Load State Last 9"),
```
Comment (degasus) [inline Source/Core/Core/HotkeyManager.cpp]: This two lines are the bug fix.

Comment (Armada651) [inline Source/Core/Core/HotkeyManager.cpp]: Should this be in the stable?

Comment (degasus) [inline Source/Core/Core/HotkeyManager.cpp]: As this is a fix, yes

Comment (degasus) [inline Source/Core/Core/HotkeyManager.cpp]: @Armada651 I think only this two lines should go to stable. I'll create a new PR for stable soon.

**Pull request merged**